Makefile

```diff
@@ -194,6 +194,26 @@ config_morse_localnet: ## Create a localnet config file to serve as a Morse gate
 		echo "##################################################################################################################"; \
 	fi
 
+###############################
+###  Localnet Make targets  ###
+###############################
+.PHONY: localnet_up
+localnet_up: ## Spins up Kind cluster for local development and brings up Tilt from file
```

Comment:
Any chance you can make this similar to what we have in `poktroll` for consistency?

```Makefile
.PHONY: localnet_up
localnet_up: check_docker_ps check_kind_context proto_regen localnet_regenesis ## Starts up a clean localnet
	tilt up
```

- Avoid inline flags
- Assume the cluster is already created
- Etc...

A couple other high level considerations:
- You can create a target called `make kind_config`  to do the things you're trying
- After you merge with main, check if there are any other better consolidations to be had.

Reply:
I have split into `localnet.mk` to handle everything localnet-related.